courses/models.py

```diff
@@ -19,6 +20,24 @@ class Program(models.Model):
     def __str__(self):
         return self.title
 
+    def get_course_price(self):
+        """
+        Returns a decimal course price attached to this program.
+
+        Note: This implementation of retrieving a course price is a naive lookup that assumes
+        all course runs and courses will be the same price for the foreseeable future.
```

Comment:
Please change `all course runs and courses will...` to `all course runs in a single program will...`